### DISCUSSION TITLE
Tasmota 9.5.0 and NodeMCU - Unable to control all GPIO

### DISCUSSION THREAD

**Discussion by checkers1173** [arendst/Tasmota#12593] [Q&A] [answered]
Hi, 

I have a NodeMCU hooked up to a DHT11 sensor and WS2812B LEDs both of which work fine. I also have three 5V relays connected (optically coupled) each of which are attached to GPIO0, 12 and 16.

The configuration is as follows:

![image](https://user-images.githubusercontent.com/30567939/124825970-31ce5580-df6c-11eb-9dc2-4dd8ae45be1d.png)

However, I can only control 1 relay out of the 3 and it's the one connected to GPIO12 (or D6).  The WebUI also looks like this. I was expecting 3 toggle buttons to appear - 1 for each relay? However, Toggle 1 controls the above relay whereas Toggle 2 actually turns the WS2812B on and off.

![image](https://user-images.githubusercontent.com/30567939/124826830-36473e00-df6d-11eb-998c-493810f14335.png)

I've tried swapping pins around i.e. trying the other GPIO's where possible - but no luck. I know the NodeMCU is capable of controlling several relays as there are examples out there.

Any ideas what might be wrong i.e. 1 - why can't i control the other two relays, and 2, why there are no corresponding Toogle buttons in the WebUI?

**Comment by barbudor** [comment, marked as answer]
You defined 3 times Relay_1
Change the index for 2 of them

**Reply by checkers1173** [reply]
Thanks. That worked and I now understand what those drop down menu's are for on the far right!
